lib/features/push_notification/presentation/listener/email_change_listener.dart

```diff
@@ -162,8 +161,10 @@ class EmailChangeListener extends ChangeListener {
         _accountId!,
         _userName!,
         state,
-        propertiesCreated: EmailUtils.getPropertiesForEmailGetMethod(_session!, _accountId!),
-        propertiesUpdated: ThreadConstants.propertiesUpdatedDefault,
```

Comment:
I still not found where the code we use null for properties of `Email/get`

Reply:
is there in `getNewReceiveEmailFromNotification`?

Reply:
In `getEmailChangesToPushNotification` of FCMRepository